analytics_logs_cloudformation.json

```diff
@@ -0,0 +1,126 @@
+{
```

Comment:
Isn't this one already checked in to the NWIS_Analytics repo?

Reply:
I do think this one might live in some kind of pipeline repo someday after a repo refactor, but should probably stick with the code that generates the data for now.

Reply:
ahh I couldn't remember! thanks, I'll remove it if I can figure out how.